### PR TITLE
lesson 1-4

### PR DESCRIPTION
**Pull request by dianayossi**
lesson 1-4

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,19 +1,22 @@
...
 import AddTodoForm from './AddTodoForm';
 import TodoList from './TodoList';
 
 function App(){
 
-  const [ newTodo, setNewTodo ] = React.useState(' ');
+  const [todoList, setTodoList] = useState([]);
+
+  const addTodo= (newTodo) => {
+    setTodoList([...todoList, {newTodo}])
```
Comment (XeniyaDob) [inline src/App.js:10]: Cool! You can remove {} for newTodo obj
`setTodoList([...todoList, newTodo])`

File: src/AddTodoForm.js [hunk 2]
```diff
@@ -1,20 +1,29 @@
...
 
+    const handleAddTodo = (event) => {
+        event.preventDefault();
+
+        onAddTodo(
+        {
+            title: todoTitle,
+            id: Date.now(),
+        }
+        )
```
Comment (XeniyaDob) [inline src/AddTodoForm.js:20]: Great job overall! Just small detail you need to add logic to reset the todoTitle state to an empty String.

**Review by XeniyaDob — APPROVED** [review]
The app is working! Please address my comments before merging to the main.

File: src/App.js (hunk 1 above)
Comment (dianayossi) [reply src/App.js:10]: Hi! If I remove the {} from the newTodo obj it breaks the app.

File: src/AddTodoForm.js (hunk 2 above)
Comment (dianayossi) [reply src/AddTodoForm.js:20]: Will do!

**Pull request merged**